engine/src/mac-window.mm

```diff
@@ -1588,6 +1588,8 @@ - (void)drawRect: (NSRect)dirtyRect
 	//////////
 	
 	MCGRegionDestroy(t_update_region);
+	
+	MCMacPlatformSyncUpdateAfterDraw(self.window.windowNumber);
```

Comment:
I take it this forces the event sifting in DoUpdate to break? (Perhaps a comment to that effect would be useful :) ).

Reply:
It does, I've added a comment.